setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# 201 and 202 are spaces after ( and before ), 501 is line length. These
+# are exceptions allowed (encouraged?) by Galaxy style guidelines.
```

Comment:
(@jmchilton you may have accidentally a word there which removes some of the meaning of the sentence: "...their their civil disobediance". Feel free to remove this comment if/when you update)